staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -84,9 +88,22 @@ type RequestDigest struct {
 // this type and cfgMeal follow the convention that the suffix
 // "Locked" means that the caller must hold the configController lock.
 type configController struct {
+	name             string // varies in tests of fighting controllers
+	clock            clock.PassiveClock
 	queueSetFactory  fq.QueueSetFactory
 	obsPairGenerator metrics.TimedObserverPairGenerator
 
+	// Puts something in the workqueue to trigger work
+	finishHandlingNotification func(workqueue.RateLimitingInterface, interface{})
+
+	// How this controller appears in an ObjectMeta ManagedFieldsEntry.Manager
+	asFieldManager string
```

Comment:
Do we really need this? Giving too many parameters to control if those aren't really needed is not the best option in my personal opinion.

Reply:
Added explanation in TestableConfig.